docs/docs-beta/docs/dagster-plus/deployment/code-locations.md

```diff
@@ -3,4 +3,237 @@ title: "Code locations"
 displayed_sidebar: "dagsterPlus"
 ---
 
-# Code locations
\ No newline at end of file
+# Code locations
+
+Separate code locations allow you to deploy different projects that still roll up into a single Dagster+ deployment with one global lineage graph. To add your first code location, follow the [getting started](/dagster-plus/getting-started) guide.
+
+This guide will cover three options for adding a new code location:
+- Adding a code location manually
+- Adding a code location in a new Git repository
+- Adding a new code location to a Git monorepo
+
+<details>
+<summary>Prerequisites</summary>
+
+1. You should have a Dagster project, we recommend setting this project up as a Python module following our [recommended project structure](/tutorial/create-new-project).
+
+2. You should know whether or not you are using Dagster Plus Serverless or Dagster Plus Hybrid. Your administrator will know that, or you can find the agent type by navigating to your Dagster+ organization, clicking on the "Deployment" page, and selecting the "Agent" tab. Dagster+ Serverless organizations will have a label "Managed by Dagster+".
+
+</details>
+
+Adding a code location follows two general steps:
+- Putting your Dagster code in a place that's accessible by your agent. For Dagster+ Serverless you can skip this step. For Dagster+ Hybrid this usually means building a Docker image with your code and pushing it to a registry.
+- Notifying Dagster+ of the new or updated code location. This will be done by using the Dagster+ Python client.
+
+Often these two steps are handled by CICD connected to your Git repository.
+
+
+## Add a new code location manually
+
+Start by installing the `dagster-cloud` Python client:
+
+```
+pip install dagster-cloud
+```
+
+Next you will want need to authenticate this Python client:
+
+1. Navigate to your Dagster+ organization. Click on your user icon in the upper right corner, then select "Organization settings". Select "Tokens" and press "Create user token". Copy the token.
+
+2. Set the following environment variables:
+
+```bash
+export  DAGSTER_CLOUD_ORGANIZATION="organization-name" # if your URL is https://acme.dagster.plus your organization name is "acme"
+export  DAGSTER_CLOUD_API_TOKEN="your-token"
+```
```

Comment:
This needs to be indented to correctly nest under this list item:

````suggestion
   ```bash
   export  DAGSTER_CLOUD_ORGANIZATION="organization-name" # if your URL is https://acme.dagster.plus your organization name is "acme"
   export  DAGSTER_CLOUD_API_TOKEN="your-token"
   ```
````

I'm not going to add comments for all other instances of this, but it's something to check for when writing.